File: app/src/main/java/org/l3ger0j/simpledimpledraw/utils/DrawPaint.java
package org.l3ger0j.simpledimpledraw.utils;

import android.graphics.Color;
import android.graphics.Paint;

public class DrawPaint extends Paint {

    public DrawPaint () {
        setColor(Color.BLACK);
        setStrokeWidth(30);
        setStrokeJoin(Paint.Join.ROUND);
        setStrokeCap(Paint.Cap.ROUND);
        setStyle(Style.STROKE);
    }
}

File: app/src/main/java/org/l3ger0j/simpledimpledraw/utils/DrawPaint.kt
package org.l3ger0j.simpledimpledraw.utils

import android.graphics.Color
import android.graphics.Paint

class DrawPaint : Paint() {
    init {
        color = Color.BLACK
        strokeWidth = 30f
        strokeJoin = Join.ROUND
        strokeCap = Cap.ROUND
        style = Style.STROKE
    }
}